lib/fs_utils/file_list.js

```diff
@@ -163,6 +164,7 @@ class FileList extends EventEmitter {
   copy(asset) {
     const path = asset.path;
     const resetCopy = (p) => {
+      if (this.disposed) { return p; }
```

Comment:
Could this be indicative of a higher level issues? In general if a file list is disposed, it shouldn't be used anywhere, and nothing should reference it